administrator/components/com_content/tmpl/article/edit.php

```diff
@@ -99,6 +99,10 @@
 		<?php if ( ! $isModal && $assoc) : ?>
 			<?php echo JHtml::_('bootstrap.addTab', 'myTab', 'associations', JText::_('JGLOBAL_FIELDSET_ASSOCIATIONS')); ?>
 			<?php echo $this->loadTemplate('associations'); ?>
+			<a class="btn btn-primary" data-toggle="modal" role="button" href="#ModalSelectArticle_auto_associations" title="Test Auto Association">
+				<span class="icon-file" aria-hidden="true"></span>Test
```

Comment:
Replace hard coded Title and Name by a Language Key like _MAKE_ASSOCIATIONS

Reply:
Got it 👌